### ISSUE TITLE
LiveTV rpc channels not working as expected on chromecast since update to 3.2.27.0

### ISSUE THREAD

**Issue by elboletaire** [MediaBrowser/Emby#2831]
Since I've updated to 3.2.27.0 there are two error cases when playing RPC live channels using chromecast (with the webplayer it works as expected):

1. When I start playing it sounds, but after a few seconds (less than 5) the sound stops playing, but the video looks good.
2. The audio doesn't stop playing, but both the audio and the video are glitchy. The video shows half of the screen with part of the image repeated, and the other half with the image stretched (photo attached; please do not consider the vertical line, I project to a pair of curtains).

BTW, playing HD channels doesn't still work either. Issue #2798 should be reopened.

#### Photo attached (of second case):

![2017-08-19 21 16 48](https://user-images.githubusercontent.com/153305/29489618-a0b025e6-8525-11e7-9db7-33cb67398e14.jpg)

#### Installation details

- **Emby server version:** docker installation (on a debian host), version 3.2.27.0.
- **Kind of LiveTV stream:** rpc channels like `rtp://@239.0.0.76:8208`
- **Emby premiere:** lifetime
- **LiveTV config:** as shown in #2798 

#### Logs

[emby logs.zip](https://github.com/MediaBrowser/Emby/files/1236595/emby.logs.zip)

**Comment by LukePulverenti** [comment]
Unfortunately this is just a hard one to test because it is specific to the input. There is no obvious problem in the log files here. if you really think it is related to a server change then I'll need you to help by installing an older version so that we can gather some comparison data.

**Comment by elboletaire** [comment]
Ofc @LukePulverenti, that's why I told you to help the first time.. I know it's not easy.

The thing is.. I use a docker installation, but AFAIK there are no tags in docker, so I cannot install an old version. How should I proceed?

**Comment by LukePulverenti** [comment]
Please try again with the new release of Emby server, thanks.

**Comment by elboletaire** [comment]
Really much better now!! Some glitches in some cases, specially at the beginning, but that's something that already happened before. Was it finally a regression from some specific version change?

I also see that you already created versioned docker tags; well done!

BTW, I still do have the issue #2650, which is not very disturbing when I'm playing, but it is when I'm recording automatically (because I cannot change the audio stream after it has been recorded, ofc).

Anyway, thank you so much Luke 😃

**Comment by LukePulverenti** [comment]
Thanks for the feedback.

**Comment by elboletaire** [comment]
Exactly the same issue after upgrading to 3.2.40.0

**Comment by LukePulverenti** [comment]
How to report a problem: https://emby.media/community/index.php?/topic/739-how-to-report-a-problem/

Thanks.

**Comment by elboletaire** [comment]
@LukePulverenti the issue is exactly the same. What do you exactly need a part from the specified in the original post? The current logs?

**Comment by LukePulverenti** [comment]
It should be resolved for the next release, thanks.

**Comment by elboletaire** [comment]
This is an issue again since a long time ago. 

![image](https://user-images.githubusercontent.com/153305/82161408-aba10c80-989c-11ea-908b-8af108ea57c3.jpg)

It was fixed, but a few versions later was broken again.

**Comment by LukePulverenti** [comment]
Hi, can we look at a new example? thanks.

**Comment by elboletaire** [comment]
What you mean a new example? I guess you refer to the logs of this case, right? Because the picture was taken yesterday :sweat_smile:. If so, I'll try to add as much information as possible in a few hours (now I'm working and can't take a look to it). Just confirm me what you mean with "a new example" and I'll try to do it :)

Edit: Oh btw, forgot to mention that the issue is only happening with the video. Sound sounds great.

**Comment by LukePulverenti** [comment]
Great, yea that's what i mean. Thanks !

**Comment by elboletaire** [comment]
Maybe I should open a new issue because there are some other problems:

- Some channels just don't start playing. I see everything in black. Here are two logs from an SD and an HD channel, just in case they may be different.
[black_SD_channel.txt](https://github.com/MediaBrowser/Emby/files/4662114/black_SD_channel.txt)
[black_HD_channel.txt](https://github.com/MediaBrowser/Emby/files/4662115/black_HD_channel.txt)
- Other channels open, and the "encoding output" appears broken, like shown in the pictures above. [cut_channel.txt](https://github.com/MediaBrowser/Emby/files/4662126/cut_channel.txt)

Looking at the logs, I can't tell a difference between those not working and those with the wrong "encoding output".

Also, when creating these logs I did realize that checking the "trace debug" checkbox in logs, makes all channels stop working. Not even those being cut are shown if I enable the trace debug. Here's the embyserver.txt file, after checking the trace checkbox and trying to play a channel (again, a channel which "works" showing the wrong encoding outputt, but stopped working with the trace debug enabled).

[embyserver.txt](https://github.com/MediaBrowser/Emby/files/4662125/embyserver.txt)

Edit: Sorry, I did forgot to add some info...

Emby version is 4.4.2 running in a docker container. Host machine is an udoo bolt V8 (a machine with an AMD Ryzen V8, with an integrated GPU Radeon Vega 8 and 8GB of RAM). Host OS is ubuntu 20.04.
